### PR TITLE
BTA-6573 - API docs update

### PR DESCRIPTION
**Pull request by mchochowski**
Adds business recipient description to ZAR payouts:
- Updates available options for legal entity types and nature of business.
- Remove list of generic bank codes, we now only validate the code is 6 digits in length   

https://azafinance.atlassian.net/browse/BTA-6573

Screenshot 2021-06-16 at 10 09 44

### CODE REVIEW

**Review by linniethepooh — APPROVED** [review]

**Review by thirunjuguna — APPROVED** [review]

**Review by thirunjuguna — CHANGES_REQUESTED** [review]
For the Business Recipient we'll need to add `contact_first_name`  and `contact_last_name`

**Review by thirunjuguna — DISMISSED** [review]

**Comment by linniethepooh** [comment]
please do no merge until I have resolved the following comments: https://github.com/bitpesa/bitpesa-api-specification/commit/37745f85f925294e399c0e422fb7e63dd57f7f2f

**Comment by thirunjuguna** [comment]
We'll need to update `entity_type` @linniethepooh

**Comment by mchochowski** [comment]
> We'll need to update `entity_type` @linniethepooh

@thirunjuguna, @linniethepooh can I help here? Is it about renaming `entity_type` to `legal_ entity_type`?

File: _docs/changelog.md [hunk 1]
```diff
@@ -7,7 +7,15 @@ permalink: /docs/changelog/
 {:toc}
 
 Current version of the API is `1`
-Current version of the SDKs are `1.13.0`
+Current version of the SDKs are `1.14.1`
+
+1.14.0
```
Comment (mchochowski) [inline _docs/changelog.md:12]:
```suggestion
1.14.1
```

File: _docs/payout-details.md [hunk 2]
```diff
@@ -553,31 +555,93 @@ For South African bank payments please use the following recipient details:
...
+- finance_and_holding_companies: Finance And Holding Companies
+- insurance: Insurance
+- business_services: Business Services
+- real_estate_development_investment: Real Estate Development Investment
+- central_state_governments: Central State Governments
+- community_services_defence_police_prisons_etc: Community Services Defence Police Prisons Etc
+- social_services_education_health_care: Social Services Education Health Care
+- personal_services_leisure_services: Personal Services - Leisure Services
+- personal_services_domestic_laundry_repairs: Personal Services - Domestic Laundry Repairs
+- personal_services_embassies_international_organisations: Personal Services - Embassies
```
Comment (mchochowski) [inline _docs/payout-details.md:640]: would be good to unify key/value order in both of these enum sets

File: _docs/changelog.md [hunk 3]
```diff
@@ -7,7 +7,15 @@ permalink: /docs/changelog/
...
+-----
+* Update `ZAR::Bank` business payout details
+
+1.14.0
+-----
+* Add support for the `ZAR::Bank` business payouts
```
Comment (cesarecamurani) [inline _docs/changelog.md:18]: Shouldn't we mention the changes for the patch? (Like in the 1.12.2 below)

Comment (cesarecamurani) [reply _docs/changelog.md:18]: Also, shouldn't we be having only 1 description of changes per release? here we have 1.14.0 twice...

Comment (linniethepooh) [reply _docs/changelog.md]: yep, it was a typo changed it to 1.14.1

Comment (cesarecamurani) [reply _docs/changelog.md]: One more nitpick... should we be more explicit in the changes description? Like for instance mentioning the `legal_entity_type` change? As for https://github.com/transferzero/api-documentation/blob/c36ff8990f7d820c1f3c9dc9bba0f83011aecc8e/_docs/changelog.md#141...

File: _docs/payout-details.md (hunk 2 above)
Comment (linniethepooh) [reply _docs/payout-details.md]: @mchochowski change made, also updated the business sender nature_of_business so they're aligned

File: _docs/changelog.md (hunk 3 above)
Comment (linniethepooh) [reply _docs/changelog.md]: change made

**Review by cesarecamurani — APPROVED** [review]

**Pull request merged**